nodejs/Dockerfile.tmpl

```diff
@@ -0,0 +1,40 @@
+FROM unocha/nodejs-base:%%UPSTREAM%%
+
+# Parse arguments for the build command.
+ARG VERSION
+ARG VCS_URL
+ARG VCS_REF
+ARG BUILD_DATE
+
+# A little bit of metadata management.
+# See http://label-schema.org/
+LABEL org.label-schema.schema-version="1.0" \
+      org.label-schema.build-date=$BUILD_DATE \
+      org.label-schema.vendor="UN-OCHA" \
+      org.label-schema.version=$VERSION \
+      org.label-schema.vcs-url=$VCS_URL \
+      org.label-schema.vcs-ref=$VCS_REF \
+      org.label-schema.name="nodejs-base" \
+      org.label-schema.description="This service provides a nodejs service platform." \
+      org.label-schema.architecture="x86_64" \
+      org.label-schema.distribution="Alpine Linux" \
+      org.label-schema.distribution-version="3.6" \
+      info.humanitarianresponse.nodejs="$NODE_VERSION"
```

Comment:
This needs to be set as env.

Reply:
Done. It is now set in nodejs-base image and used on nodejs and nodejs-builder